src/shared/scrapers/AUS/SA/index.js

```diff
@@ -17,14 +31,29 @@ const scraper = {
   type: 'paragraph',
   url:
     'https://www.sahealth.sa.gov.au/wps/wcm/connect/public+content/sa+health+internet/health+topics/health+topics+a+-+z/covid+2019/latest+updates/confirmed+and+suspected+cases+of+covid-19+in+south+australia',
-  async scraper() {
-    const $ = await fetch.page(this.url);
-    const paragraph = $('.middle-column p:first-of-type').text();
-    const { casesString } = paragraph.match(/been (?<casesString>\d+) confirmed cases/).groups;
-    return {
-      state: scraper.state,
-      cases: parse.number(casesString)
-    };
+  scraper: {
+    '0': async function() {
+      const $ = await fetch.page(this.url);
+      const paragraph = $('.middle-column p:first-of-type').text();
+      const { casesString } = paragraph.match(/been (?<casesString>\d+) confirmed cases/).groups;
+      return {
+        state: scraper.state,
+        cases: parse.number(casesString)
+      };
+    },
+    '2020-3-27': async function() {
+      scraper.type = 'table';
```

Comment:
Is there a better way to change type per date of scraper?

Reply:
Nah, just `this.type` (`scraper.type` works ofc, too)